cityofaustin/atd-vz-data
# Align CRIS import to the editable fields of the VZE's crash page

## Associated issues

This PR is related to this work: https://github.com/cityofaustin/atd-data-tech/issues/16157.

## Testing

Local:
* Create a `env` file for the CRIS import with the 3 usual suspect entries to enable 1PW API access
```
OP_API_TOKEN=
OP_CONNECT=
OP_VAULT_ID=
```
* Spin up the DB + graphql endpoint
* Replicate production data into the DB
* Get a handle on what's inside the changes table, with something like this:
```
select count(*)
from atd_txdot_changes;
```
* Execute:
```
docker compose run cris-import
```
* Inspect the changes table and the fact tables to the CSVs that were extracted and see if you're happy with the results.

---
#### Ship list
